rllib/core/models/tf/encoder.py

```diff
@@ -27,49 +30,17 @@
 _, tf, _ = try_import_tf()
 
 
-class TfMLPEncoder(Encoder, TfModel):
```

Comment:
Just re-sorted this class.

Reply:
Cool. Since the diff becomes quite weird with this, I'll only look at the LSTM code.